src/source/Dtls/Dtls.h

```diff
@@ -38,12 +38,25 @@ typedef enum {
    SRTP_PROFILE_AES128_CM_HMAC_SHA1_32 = SRTP_AES128_CM_SHA1_32,
 } SRTP_PROFILE;
 
-// Callback that is fired when Dtls Server wishes to send packet
+typedef enum {
+    NEW,
```

Comment:
Are we going to be exposing/handling any other state other than CLOSED? In which case we need some mapping between these states and the PC states.

Reply:
the mapping is done in the dtls state changed callback.